dsc/configurations.md

```diff
@@ -61,7 +61,7 @@ Configuration MyDscConfiguration {
 		}
 	}
 }
-MyDscConfiguration
+MyDscConfiguration -ComputerName "MyComputer"
```

Comment:
I think it's better to use this syntax:

```powershell
MyDscConfiguration -ComputerName <MyComputer>
```

Reply:
Even inside a comment-block example? I don't agree, this should be copy/paste, change your values.

Reply:
The quotes to me imply they are needed.  The angle brackets is pretty universally accepted as meaning a required parameter value.

Reply:
Done